src/components/SendTransaction/CreateTransaction.tsx

```diff
@@ -288,6 +305,9 @@ export const CreateTransaction = ({
         ) {
           message =
             'Stellar address or public key is invalid. Public keys are uppercase and begin with letter "G."';
+        } else if (isAccountMalicious) {
+          message =
+            "This account has been flagged as being potentially malicious.";
```

Comment:
if something is `malicious`, show an error message and don't allow form submission